tools/js-sdk-release-tools/src/llc/generateRLCInPipeline/generateRLCInPipeline.ts

```diff
@@ -243,7 +243,7 @@ export async function generateRLCInPipeline(options: {
         logger.info(`Start to run command 'node common/scripts/install-run-rush.js pack --to ${packageName} --verbose'.`);
         execSync(`node common/scripts/install-run-rush.js pack --to ${packageName} --verbose`, {stdio: 'inherit'});
         if (!options.skipGeneration) {
-            const changelog = await generateChangelogAndBumpVersion(packagePath);
+            const changelog = await generateChangelogAndBumpVersion(relativePackagePath);
```

Comment:
just curious where to find this issue? packagePath is for absolute path but relativePackagePath  is for relative one?

can we test in our side?

Reply:
yes, forget to add test case: https://github.com/test-repo-billy/azure-rest-api-specs/pull/3557

Reply:
this caused by manual test by me. generateChangelogAndBumpVersion always use relative package path